### ISSUE TITLE
14.0 Open Timeoff calendar with Arabic language

### ISSUE THREAD

**Issue by abdallamhafeez** [odoo/odoo#63304]
Impacted versions:
14.0
Steps to reproduce:
- Activate the Arabic langauge.
- Open Timeoff app from home screen.

Current behavior:
Appear below error
```
خطأ:
Odoo Server Error

Traceback (most recent call last):
  File "/data/build/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
    result = request.dispatch()
  File "/data/build/odoo/odoo/http.py", line 683, in dispatch
    result = self._call_function(**self.params)
  File "/data/build/odoo/odoo/http.py", line 359, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/data/build/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/data/build/odoo/odoo/http.py", line 347, in checked_call
    result = self.endpoint(*a, **kw)
  File "/data/build/odoo/odoo/http.py", line 912, in __call__
    return self.method(*args, **kw)
  File "/data/build/odoo/odoo/http.py", line 531, in response_wrap
    response = f(*args, **kw)
  File "/data/build/odoo/addons/web/controllers/main.py", line 1377, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/data/build/odoo/addons/web/controllers/main.py", line 1369, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/data/build/odoo/odoo/api.py", line 392, in call_kw
    result = _call_kw_model(method, model, args, kwargs)
  File "/data/build/odoo/odoo/api.py", line 365, in _call_kw_model
    result = method(recs, *args, **kwargs)
  File "/data/build/odoo/addons/hr_holidays/models/hr_leave.py", line 1248, in get_unusual_days
    dfrom = datetime.combine(fields.Date.from_string(date_from), time.min).replace(tzinfo=UTC)
  File "/data/build/odoo/odoo/fields.py", line 1771, in to_date
    return datetime.strptime(value, DATE_FORMAT).date()
  File "/usr/lib/python3.6/_strptime.py", line 565, in _strptime_datetime
    tt, fraction = _strptime(data_string, format)
  File "/usr/lib/python3.6/_strptime.py", line 362, in _strptime
    (data_string, format))
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/data/build/odoo/odoo/http.py", line 639, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/data/build/odoo/odoo/http.py", line 315, in _handle_exception
    raise exception.with_traceback(None) from new_cause
ValueError: time data '٢٠٢٠-١١-٢٧' does not match format '%Y-%m-%d'
```

**Comment by RachidAlassir** [comment]
I think it was fixed in #63833

**Comment by Yenthe666** [comment]
Yes indeed, that should fix it.
